src/save-key.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components/macro';
+import { connect } from 'react-redux'
+
+import { saveKey } from './actions';
+
+import theme from './theme';
+import mediaQueries from './media-queries';
+
+import { Container } from './components/container';
+import { Row } from './components/row';
+import { FormInput } from './components/form-input';
+import { Button } from './components/button';
+
+import './save-key.scss';
+
+const Col = styled.div`
+    width: 100%;
+    padding-left: ${theme.gutter};
+    padding-right: ${theme.gutter};
+    text-align: center;
+`;
+Col.displayName = 'Col';
+
+
+const OuterCol = styled(Col)`
+    height: 100%;
+    margin: auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+
+    form {
+        width: 100%;
+        display: block;
+        max-width: ${mediaQueries.min.iphone6};
+    }
+`;
+OuterCol.displayName = 'OuterCol';
+
+export const SaveKey = (props) => {
+    const [fieldValue, setFieldValue] = useState('');
```

Comment:
Hooks!!! Fun! 

Would it be super difficult to convert the other class based components to use functional components with hooks? One thing I'd be looking for in a code review is code consistency. Though I don't think it'd be a deal breaker 😄